state-synchronizer/src/coordinator.rs

```diff
@@ -188,9 +179,9 @@ impl<T: ExecutorProxyTrait> SyncCoordinator<T> {
     }
 
     fn target_version(&self) -> u64 {
```

Comment:
nit: mind to retire this function completely and just make call site use SyncRequest directly?